app/proguard-rules.pro

```diff
@@ -39,12 +35,11 @@
 }
 -keepnames class * { @icepick.State *;}
 
-# Rules for OkHttp. Copy paste from https://github.com/square/okhttp
+## Rules for OkHttp. Copy paste from https://github.com/square/okhttp
```

Comment:
What's the idea behind the double hash?

Reply:
I think it would be better if you just leave the ``#`` and a a line above 
```
-keepclassmembers class * implements java.io.Serializable
```

that says something like ``# keep required serialized stuff``

Reply:
I wasn't intending to add any note to that, I just wanted to section off the OKHttp stuff from the rest of the file. I guess I could add a note for that though, if you want.